src/terraform/structure/tf_taggable.go

```diff
@@ -1,679 +1,79 @@
-	"azurerm_vmware_private_cloud",
-	"azurerm_vpn_gateway",
-	"azurerm_vpn_server_configuration",
-	"azurerm_vpn_site",
-	"azurerm_web_application_firewall_policy",
-	"azurerm_windows_virtual_machine",
-	"azurerm_windows_virtual_machine_scale_set",
-	"google_active_directory_domain",
-	"google_assured_workloads_workload",
-	"google_bigquery_dataset",
-	"google_bigquery_job",
-	"google_bigquery_table",
-	"google_bigtable_instance",
-	"google_cloud_identity_group",
-	"google_cloudfunctions_function",
-	"google_composer_environment",
-	"google_compute_disk",
-	"google_compute_image",
-	"google_compute_instance",
-	"google_compute_instance_from_template",
-	"google_compute_instance_template",
-	"google_compute_region_disk",
-	"google_compute_snapshot",
-	"google_dataflow_job",
-	"google_dataproc_cluster",
-	"google_dataproc_job",
-	"google_dataproc_workflow_template",
-	"google_dialogflow_cx_intent",
-	"google_dns_managed_zone",
-	"google_eventarc_trigger",
-	"google_filestore_instance",
-	"google_game_services_game_server_cluster",
-	"google_game_services_game_server_config",
-	"google_game_services_game_server_deployment",
-	"google_game_services_realm",
-	"google_gke_hub_membership",
-	"google_healthcare_consent_store",
-	"google_healthcare_dicom_store",
-	"google_healthcare_fhir_store",
-	"google_healthcare_hl7_v2_store",
-	"google_kms_crypto_key",
-	"google_memcache_instance",
-	"google_ml_engine_model",
-	"google_monitoring_notification_channel",
-	"google_network_management_connectivity_test",
-	"google_network_services_edge_cache_keyset",
-	"google_network_services_edge_cache_origin",
-	"google_network_services_edge_cache_service",
-	"google_notebooks_instance",
-	"google_privateca_ca_pool",
-	"google_privateca_certificate",
-	"google_privateca_certificate_authority",
-	"google_project",
-	"google_pubsub_subscription",
-	"google_pubsub_topic",
-	"google_redis_instance",
-	"google_secret_manager_secret",
-	"google_spanner_instance",
-	"google_storage_bucket",
-	"google_tpu_node",
-	"google_vertex_ai_dataset",
-	"google_workflows_workflow",
+import (
+	"strings"
+
+	"github.com/ahmetb/go-linq/v3"
+	tfjson "github.com/hashicorp/terraform-json"
+	awsv2 "github.com/lonegunmanb/terraform-aws-schema/v2/generated"
+	awsv3 "github.com/lonegunmanb/terraform-aws-schema/v3/generated"
+	awsv4 "github.com/lonegunmanb/terraform-aws-schema/v4/generated"
+	awsv5 "github.com/lonegunmanb/terraform-aws-schema/v5/generated"
+	azurev2 "github.com/lonegunmanb/terraform-azurerm-schema/v2/generated"
+	azurev3 "github.com/lonegunmanb/terraform-azurerm-schema/v3/generated"
+	googlev2 "github.com/lonegunmanb/terraform-google-schema/v2/generated"
+	googlev3 "github.com/lonegunmanb/terraform-google-schema/v3/generated"
+	googlev4 "github.com/lonegunmanb/terraform-google-schema/v4/generated"
+)
+
+var TfTaggableResourceTypes []string
+
+func init() {
+	linq.From(previousTaggableTypes(awsv4.Resources, isTaggableType, awsv2.Resources, awsv3.Resources, awsv4.Resources)).
```

Comment:
```suggestion
	linq.From(previousTaggableTypes(awsv4.Resources, isTaggableType, awsv2.Resources, awsv3.Resources)).
```
it looks like you have `aws4` twice in the line

Reply:
My bad...